iocore/eventsystem/UnixEventProcessor.cc

```diff
@@ -32,399 +32,229 @@
 #include "ts/ink_defs.h"
 #include "ts/hugepages.h"
 
+#include <algorithm>
+
 /// Global singleton.
 class EventProcessor eventProcessor;
 
-class ThreadAffinityInitializer : public Continuation
```

Comment:
As it is ... BTW .. the main purpose I wrote what I did is (1) to prevent allocating EThread inside the EventProcessor code.   Huge and crazy Factory/Object creation is *never* something to do deep down.   I can see how EThread has been way way way overburdened because it's so stuck there.

It's far easier to let it get customized (and it already has been elsewhere) to a lighter object type.  That means new-ing it elsewhere.   MOST importantly (for Jemalloc) in the correct Arena and on the correct NUMA nodes.

So I gave a try at shrinking it down using lambdas.  Worked great.   It's able to be seperated out, though.